### DISCUSSION TITLE
hellpot issue??

### DISCUSSION THREAD

**Discussion by d3athg0d** [telekom-security/tpotce#1011] [Q&A] [answered]
i was trying the hellpot however it seems to me that it is not working.

when i visit the url under my tpot IP, it show "Not Found" instead of the endless loading webpage shown in https://github.com/carlmjohnson/heffalump

the kibana page also do not have any information capture
 am i doing anything wrongly?

![error_hellpot](https://user-images.githubusercontent.com/78822442/153751066-efadefe1-ddb0-40d0-8388-c53fd3cb3406.png)
![error_hellpot1](https://user-images.githubusercontent.com/78822442/153751106-c2b633cc-d292-4798-bb47-7aac91ffffb1.png)
![error_hellpot2](https://user-images.githubusercontent.com/78822442/153751167-01defa36-62f5-4dcb-8825-3645fa38a8d3.png)

**Comment by t3chn0m4g3** [comment, marked as answer]
Works as expected. Hellpot is not configured to behave like Heffalump and our config won't serve an endless page on the root path.

**Reply by d3athg0d** [reply]
glad to learn that is it working according to design.
appreciate your reply. thanks.
